### ISSUE TITLE
Errors while trying to run docker containers

### ISSUE THREAD

**Issue by exelents** [sld/convai-bot-1337#32]
Hello.
I'm trying to run your bot on my computer, but when I execute 
`docker-compose -f docker-compose.yml -f json_api.yml up`
and comp runs docker containers with components of bot I see many errors ( see logs in attachment ) in console.
Can anyone help me, give some recommendations?

[convbot-logs.log.zip](https://github.com/sld/convai-bot-1337/files/1597721/convbot-logs.log.zip)

**Comment by exelents** [comment]
Okay, there was a problem, no enough disk space, so some models hadn't copied.
Now ./setup.sh had copied them well, but some modules does't found their models again. For example:
`opennmtchitchat_1          | tput: No value for $TERM and no -T specified
opennmtchitchat_1          | translation_server.lua: invalid argument for option -model: if set, the file must exist
opennmtchitchat_1          | Try 'translation_server.lua -h' for more information, or visit the online documentation at http://opennmt.net/OpenNMT/.
`
There are no model files in folder opennmt_chitchat/data and no commands in ./setup.sh to copy any file there.

Another problem:
`intent_classifier_1        | [2018-01-03 03:54:22 +0000] [1] [CRITICAL] WORKER TIMEOUT (pid:15)
intent_classifier_1        | [2018-01-03 03:54:22 +0000] [15] [INFO] Worker exiting (pid: 15)
intent_classifier_1        | [2018-01-03 03:54:22 +0000] [21] [INFO] Booting worker with pid: 21
intent_classifier_1        | Loading embeddings: ./data/glove.6B.100d.txt
intent_classifier_1        | [2018-01-03 03:54:52 +0000] [1] [CRITICAL] WORKER TIMEOUT (pid:21)
intent_classifier_1        | [2018-01-03 03:54:52 +0000] [21] [INFO] Worker exiting (pid: 21)
intent_classifier_1        | [2018-01-03 03:54:53 +0000] [27] [INFO] Booting worker with pid: 27
intent_classifier_1        | Loading embeddings: ./data/glove.6B.100d.txt`
There are file glove.6B.100d.txt in folder intent_classifier/data but it doesn't loaded. I have no idea why.

There are some other errors with parsing some xml like:
`alice_1                    | PARSE ERROR: Unexpected <category> tag (line 100, column 1)
alice_1                    | PARSE ERROR: Unexpected </learn> tag (line 108, column 0)
alice_1                    | PARSE ERROR: Unexpected </li> tag (line 110, column 0)
alice_1                    | PARSE ERROR: Unexpected </condition> tag (line 128, column 0)
alice_1                    | PARSE ERROR: Unexpected </template> tag (line 129, column 0)
alice_1                    | PARSE ERROR: Unexpected </category> tag (line 130, column 0)
alice_1                    | PARSE ERROR: Unexpected <category> tag (line 158, column 1)
alice_1                    | PARSE ERROR: Unexpected </learn> tag (line 166, column 0)
alice_1                    | PARSE ERROR: Unexpected </li> tag (line 168, column 0)
alice_1                    | PARSE ERROR: Unexpected </condition> tag (line 173, column 0)
alice_1                    | PARSE ERROR: Unexpected </template> tag (line 174, column 0)
alice_1                    | PARSE ERROR: Unexpected </category> tag (line 175, column 0)
`
Look for new logs for more information:
[convbot-logs.log](https://github.com/sld/convai-bot-1337/files/1599378/convbot-logs.log)

**Comment by exelents** [comment]
First problem with opennmtchitchat_1 was defeated with copying model file from data folder. In your ./setup.sh you need to add such command:
`cp data/CPU_epoch5_14.62.t7 opennmt_chitchat/data/`
But intent_classifier_1 still doesn't load GloVe vectors, it falls by timeout.  And conteiner alice_1 have problems with parsing tags

**Comment by sld** [comment]
Thanks! I added opennmt_chitchat/data/ copying just now: https://github.com/sld/convai-bot-1337/commit/3e35aac28c036bade84bf61063f4ab9286318ed1

It is ok that Alice have problems with parsing some tags. It affects only some minor skills of Alice.

I am also having the same error with intent_classifier. But it automatically reloads after TIMEOUT and everything works.

Try to run the test https://github.com/sld/convai-bot-1337#running-the-tests to check if everything works: `python dialog_tracker/tests/test_json_api.py http://0.0.0.0:5000`

**Comment by exelents** [comment]
Second problem with  intent_classifier_1 was defeated when I have set timeout ( -t 120 ) in module run command. This command should run like:
` gunicorn -w 1 -t 120 -b 0.0.0.0:3000 server:app # Only 1 worker, beacuse a lot of ram required`
Maybe before it didn't have enough time to load all vectors.
So there are only errors with parsing some tags in alice_1 container, but I don't know does it has any influence on the bot behaviour.

I tried to run test_json_api and it works, at least it no any errors there. So it seems like everything works. I'll try to test your bot futher.
